auditbeat/magefile.go

```diff
@@ -162,3 +162,12 @@ func Dashboards() error {
 func Docs() {
 	mg.Deps(auditbeat.ModuleDocs, auditbeat.FieldDocs)
 }
+
+// UnitTestFIPS runs unit tests with go 1.24 GODEBUG=fips140=only flag set and filters out expected failures.
```

Comment:
Does this require the microsoft/go fork?

Reply:
Nope, this just uses Go 1.24.
These tests are to ensure that no non-compliant code is added by accident